src/Plugin/Plugins/BTCQuotePlugin.h

```diff
@@ -0,0 +1,253 @@
+     */
+    BTCQuotePlugin(const String& name, uint16_t uid) :
+        Plugin(name, uid),
+        m_textCanvas(nullptr),
+        m_iconCanvas(nullptr),
+        m_bitmapWidget(),
+        m_textWidget("\\calign?"),
+        m_btc_chf_xrate(-1),
+        m_btc_usd_xrate(-1),
+        m_httpResponseReceived(false),
+        m_relevantResponsePart(""),
+        m_url(),
+        m_xMutex(nullptr),
+        m_requestTimer()
+    {
+        /* Move the text widget one line lower for better look. */
+        m_textWidget.move(0, 1);
+
+        m_xMutex = xSemaphoreCreateMutex();
+    }
+
+    /**
+     * Destroys the plugin.
+     */
+    ~BTCQuotePlugin()
+    {
+        if (nullptr != m_iconCanvas)
+        {
+            delete m_iconCanvas;
+            m_iconCanvas = nullptr;
+        }
+
+        if (nullptr != m_textCanvas)
+        {
+            delete m_textCanvas;
+            m_textCanvas = nullptr;
+        }
+
+        if (nullptr != m_xMutex)
+        {
+            vSemaphoreDelete(m_xMutex);
+            m_xMutex = nullptr;
+        }
+    }
+
+    /**
+     * Plugin creation method, used to register on the plugin manager.
+     *
+     * @param[in] name  Plugin name
+     * @param[in] uid   Unique id
+     *
+     * @return If successful, it will return the pointer to the plugin instance, otherwise nullptr.
+     */
+    static IPluginMaintenance* create(const String& name, uint16_t uid)
+    {
+        return new BTCQuotePlugin(name, uid);
+    }
+
+    /**
+     * This method will be called in case the plugin is set active, which means
+     * it will be shown on the display in the next step.
+     *
+     * @param[in] gfx   Display graphics interface
+     */
+    void active(IGfx& gfx) final;
+
+    /**
+     * This method will be called in case the plugin is set inactive, which means
+     * it won't be shown on the display anymore.
+     */
+    void inactive() final;
+
+    /**
+     * Update the display.
+     * The scheduler will call this method periodically.
+     *
+     * @param[in] gfx   Display graphics interface
+     */
+    void update(IGfx& gfx) final;
+
+   /**
+     * Stop the plugin.
+     * Overwrite it if your plugin needs to know that it will be uninstalled.
+     */
+    void stop() final;
+
+    /**
+     * Start the plugin.
+     * Overwrite it if your plugin needs to know that it was installed.
+     */
+    void start() final;
+
+    /**
+     * Process the plugin.
+     * Overwrite it if your plugin has cyclic stuff to do without being in a
+     * active slot.
+     */
+    void process(void) final;
+
+private:
+
+    /**
+     * Icon width in pixels.
+     */
+    static const int16_t    ICON_WIDTH          = 8;
+
+    /**
+     * Icon height in pixels.
+     */
+    static const int16_t    ICON_HEIGHT         = 8;
+
+    /**
+     * Image path within the filesystem.
+     */
+    static const char*      BTC_USD_IMAGE_PATH;
+    static const char*      BTC_CHF_IMAGE_PATH;
```

Comment:
Doxygen description is missing. Maybe describe whats the difference between both.